symbology/render-multilayer-symbols/src/main/java/com/esri/samples/render_multilayer_symbols/RenderMultilayerSymbolsSample.java

```diff
@@ -0,0 +1,459 @@
+      hatchFillSymbolLayer.setSeparation(9);
+      symbolLayerArray[i] = hatchFillSymbolLayer;
+    }
+    symbolLayerArray[symbolLayerArray.length - 1] = strokeForOutline;
+    // create a multilayer polygon symbol with symbol layers
+    var multilayerPolygonSymbol = new MultilayerPolygonSymbol(List.of(symbolLayerArray));
+
+    // create a polygon graphic with geometry using the symbol created above, and add it to the graphics overlay
+    var graphic = new Graphic(polygonBuilder.toGeometry(), multilayerPolygonSymbol);
+    graphicsOverlay.getGraphics().add(graphic);
+  }
+
+  /**
+   * Creates a complex point from multiple symbol layers and a provided geometry.
+   * @param complexPointGeometry A base geometry upon which other symbol layers are drawn.
+   */
+  private void addComplexPoint(Geometry complexPointGeometry) {
+    // create marker layers for complex point
+    var orangeSquareVectorMarkerLayer = getLayerForComplexPoint(Color.ORANGE, Color.BLUE, 11);
+    var blackSquareVectorMarkerLayer = getLayerForComplexPoint(Color.BLACK, Color.ORANGERED, 6);
+    var purpleSquareVectorMarkerLayer = getLayerForComplexPoint(Color.TRANSPARENT, Color.PURPLE, 14);
+
+    // set anchors for marker layers
+    orangeSquareVectorMarkerLayer.setAnchor(new SymbolAnchor(-4, -6, SymbolAnchor.PlacementMode.ABSOLUTE));
+    blackSquareVectorMarkerLayer.setAnchor(new SymbolAnchor(2, 1, SymbolAnchor.PlacementMode.ABSOLUTE));
+    purpleSquareVectorMarkerLayer.setAnchor(new SymbolAnchor(4, 2, SymbolAnchor.PlacementMode.ABSOLUTE));
+
+    // create a yellow hexagon with a black outline
+    SolidFillSymbolLayer yellowFillLayer = new SolidFillSymbolLayer(ColorUtil.colorToArgb(Color.YELLOW));
+    SolidStrokeSymbolLayer blackOutline = new SolidStrokeSymbolLayer(2, ColorUtil.colorToArgb(Color.BLACK),
+      List.of(new DashGeometricEffect()));
+    VectorMarkerSymbolElement hexagonVectorElement = new VectorMarkerSymbolElement(complexPointGeometry,
+      new MultilayerPolylineSymbol(List.of(yellowFillLayer, blackOutline)));
+    VectorMarkerSymbolLayer hexagonVectorMarkerLayer = new VectorMarkerSymbolLayer(List.of(hexagonVectorElement));
+    hexagonVectorMarkerLayer.setSize(35);
+
+    // create the multilayer point symbol
+    var multilayerPointSymbol = new MultilayerPointSymbol(List.of(
+      hexagonVectorMarkerLayer,
+      orangeSquareVectorMarkerLayer,
+      blackSquareVectorMarkerLayer,
+      purpleSquareVectorMarkerLayer
+    ));
+
+    // create the multilayer point graphic using the symbols created above
+    Graphic complexPointGraphic = new Graphic(new Point(130, 20, SpatialReferences.getWgs84()), multilayerPointSymbol);
+    graphicsOverlay.getGraphics().add(complexPointGraphic);
+  }
+
+  /**
+   * Creates a symbol layer for use in the composition of a complex point.
+   *
+   * @param fillColor Fill colour of the symbol
+   * @param outlineColor Outline colour of the symbol
+   * @param size size of the symbol
+   * @return something idk
```

Comment:
🤣 uh ohs, something very straight forward here will do
```suggestion
   * @return the vector marker symbol layer
```

Reply:
ooft I did not leave that in deary me 🙃